dvc/state.py

```diff
@@ -449,6 +449,8 @@ def remove_unused_links(self, used):
                 remove(path)
                 unused.append(relpath)
 
-        for relpath in unused:
-            cmd = "DELETE FROM {} WHERE path = ?".format(self.LINK_STATE_TABLE)
-            self._execute(cmd, (relpath,))
+        for chunk_unused in chunk(unused, 100):
```

Comment:
So why 100 then if max is 999? Also, don't hardcode these, maybe we could reference SQLITE_MAX_VARIABLE_NUMBER somehow?

Reply:
I am unable to find this constant/ anything that looks like this constant inside sqlite package

Reply:
I updated with 999